test/read.spec.ts

```diff
@@ -65,4 +65,29 @@ describe("read .nodesecurerc", () => {
       strategy: "none"
     });
   });
+
+  it("should return the default value 'null' when the memoize option is not declared ", async() => {
+    await read(location, { createIfDoesNotExist: true });
+    expect(memoized()).to.eq(null);
+  });
+
+  it("should return the configuration passed in parameter", async() => {
+    await read(location, { createIfDoesNotExist: true, memoize: true });
+    expect(memoized()).to.deep.equal(generateDefaultRC());
+  });
+
+  it("should merge with the previous stored payload", async() => {
+    const payload = {
+      port: "3306",
+      provider: "mysql",
+      address: "localhost"
+    } as any;
```

Comment:
No need to cast as any here I think (like in others test we can create a Partial<RC>).